### PR TITLE
[esp/scene] add initial support for Replica_v1 semantic information

### PR DESCRIPTION
**Pull request by msbaines**
## Motivation and Context

Initial support for Replica semantic information.

## How Has This Been Tested
Testing semantic information using WebGL demo.

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/esp/gfx/Simulator.cpp [hunk 1]
```diff
@@ -139,13 +139,23 @@ void Simulator::reconfigure(const SimulatorConfiguration& cfg) {
...
-  if (io::exists(houseFilename)) {
-    scene::SemanticScene::loadMp3dHouse(houseFilename, *semanticScene_);
-  }
-
-  // also load SemanticScene for SUNCG house file
-  if (sceneInfo.type == assets::AssetType::SUNCG_SCENE) {
-    scene::SemanticScene::loadSuncgHouse(sceneFilename, *semanticScene_);
+  switch (sceneInfo.type) {
+    case assets::AssetType::INSTANCE_MESH:
+      houseFilename = io::parentPath(houseFilename) + "info_semantic.json";
```
Comment (erikwijmans) [inline src/esp/gfx/Simulator.cpp]: https://doc.magnum.graphics/corrade/namespaceCorrade_1_1Utility_1_1Directory.html#a222baead7128ec0cadc433e3a8c0059f

Comment (erikwijmans) [reply src/esp/gfx/Simulator.cpp]: Then you can use `Cr::Directory::join` to combine the path with the `info_semantic.json` name.

Comment (msbaines) [reply src/esp/gfx/Simulator.cpp]: Done.

**Comment by msbaines** [comment]
All feedback should now be addressed. Verified that semantic labels are working well (thanks to @erikwijmans helping fix a bug in the javascript side).

**Pull request merged**